ignition/modules/groth16Verifiers.ts

```diff
@@ -31,3 +31,8 @@ export const Groth16VerifierStateTransitionModule = buildModule(
     return { verifier: g16VerifierStateTransition };
   },
 );
+
+export const Groth16VerifierAuthV2Module = buildModule("Groth16VerifierAuthV2Module", (m) => {
+  const verifier = m.contract(contractsInfo.GROTH16_VERIFIER_AUTH_V2.name);
+  return { verifier };
+});
```

Comment:
You need to change the const and module name here with `Groth16VerifierAuthV2WrapperModule` and properly return the `wrapper`
```suggestion
export const Groth16VerifierAuthV2WrapperModule = buildModule("Groth16VerifierAuthV2WrapperModule", (m) => {
  const g16VerifierAuthV2Wrapper = m.contract(contractsInfo.GROTH16_VERIFIER_AUTH_V2.name);
  return { wrapper: g16VerifierAuthV2Wrapper };
});
```

Reply:
fixed